### ISSUE TITLE
The default app created by Quilt is not responsive

### ISSUE THREAD

**Issue by adriancorcoran** [Shopify/quilt#1086]
## Overview

**TLDR; the default app created by Quilt is not responsive and needs a component added from another package library**

---

I have used Quilt to create the basic React app but the design is not responsive - after some digging I see there is the Responsive component in `@shopify/react-html` which needs to be added.

* [Doc](https://github.com/Shopify/quilt/tree/32f04a86d8ae0f585e96dda5160ffb152073587e/packages/react-html#responsive-)
* [Example](https://github.com/Shopify/web-rails-proving-ground/blob/master/app/ui/foundation/App/components/Metadata/Metadata.tsx)

## Type

- [ ] New feature
- [x] Changes to existing features

## Motivation

> What inspired this feature request? What problems were you facing,
> or what else makes you think this should be in `quilt`?

From [discussion on Slack](https://shopify.slack.com/archives/CCNRSKLTH/p1570042355214700) I can see there are some reasons for not including this be default in some instances, however I’m sure it will be a necessary step most devs would need to take once the app is up and running. 

To save on support debt for you folks (and time spent digging for the answer to a basic feature for the dev), could I suggest a few solutions?

1. Add it in by default into the app

1. Maybe it could be a setup question in the CLI? Would pave the way for more configuration questions as well for different environments / teams / requirements?

1. Or even just adding the package include line and component into the basic app, but commented out with an explanation note - similar to the default Rails config files?

TIA :)

---

## Checklist

- [x] Please delete the labels section before submitting your issue
- [x] I have described this issue in a way that is actionable (if possible)

**Comment by adriancorcoran** [comment]
Keeping this open

**Comment by adriancorcoran** [comment]
Letting this go as I now have my own standard set of code add ons after creating a new Quilt app. If it is important to others, then feel free to reopen :)
